src/client/datascience/kernel-launcher/kernelProcess.ts

```diff
@@ -144,7 +148,7 @@ export class KernelProcess implements IKernelProcess {
             // Make sure to dispose if we never get a heartbeat
             this.dispose().ignoreErrors();
             traceError('Timed out waiting to get a heartbeat from kernel process.');
-            throw new Error('Timed out waiting to get a heartbeat from kernel process.');
+            throw new Error(localize.DataScience.kernelTimeout().format(stderr, Commands.ViewJupyterOutput));
```

Comment:
I don't think we shoudl throw an error like this with instructions in the error mssage.
This means were just going to do a `error.toString()` and display it in the messagebox.

We should instead handle the error and display a meaningful message with details of the error.